net-irc/anope/anope-2.0.7.ebuild

```diff
@@ -0,0 +1,155 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="Anope IRC Services"
+HOMEPAGE="https://anope.org"
+SRC_URI="https://github.com/anope/anope/releases/download/${PV}/${P}-source.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="chanstats cs-fantasy-stats cs-fantasy-top gnutls irc2sql ldap
+	ldap-auth ldap-oper mysql pcre posix sql-auth sql-log sql-oper
+	sqlite ssl tre"
+
+REQUIRED_USE="
+	chanstats? ( mysql )
+	cs-fantasy-stats? ( chanstats mysql )
+	cs-fantasy-top? ( chanstats mysql )
+	irc2sql? ( mysql )
+	ldap-auth? ( ldap )
+	ldap-oper? ( ldap )
+	sql-auth? ( || ( mysql sqlite ) )
+	sql-log? ( mysql )
+	sql-oper? ( mysql )"
+
+BDEPEND="sys-devel/gettext"
+DEPEND="${BDEPEND}
```

Comment:
This looks wrong. This says "We need gettext at the host system, which is building anope to execute something during build but we also need gettext at build time because we are going to link..."

Reply:
gettext is needed at buildtime to generate .po files so I think this means BDEPEND.

I read the GNU gettext [documentation](https://www.gnu.org/software/gettext/manual/gettext.html), which says:
"Note that on GNU systems, you don’t need to link with libintl because the gettext library functions are already contained in GNU libc. That is all you have to change."

This implies that we need to be careful about non-glibc systems. I did some research and it looks like at link time, we need the libintl headers, so this looks like a DEPEND on virtual/libintl is needed, which should sort e.g. musl boxes.

Reply:
Okay, have a look now.

Reply:
Okay, great. Does it look right now?